build/Targets/VSL.Versions.targets

```diff
@@ -73,7 +73,7 @@
   </Choose>
 
   <!-- Returns the current build version. Used in .vsixmanifests to substitute our build version into them -->
-  <Target Name="GetBuildVersion" Outputs="$(BuildVersion)" />
+  <Target Name="GetBuildVersion" Outputs="$(RoslynSemanticVersion).$(BuildNumberPart1.Trim())$(BuildNumberPart2.Trim())" />
```

Comment:
They were here long before my change, the parts are just x.y and they are put together differently in some scenarios (x-y for nuget per build pre-release, xy for vsix version, etc)